### PR TITLE
 Feat:    added toast notification on copy sharing link

### PR DESCRIPTION
**Pull request by AnshuPathak-88825**
solved issue #14489 
it is working perfectly

https://github.com/jitsi/jitsi-meet/assets/78332454/7fc7f5a7-cca7-4dac-9878-b558d7fa2489

### CODE REVIEW

File: react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx [hunk 1]
```diff
@@ -76,6 +81,9 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS }: IPro
      * @returns {void}
      */
     function _onCopyText() {
+        dispatch(showSuccessNotification({
+            titleKey: "Copied meeting link"
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx:85]: This needs to be translated.

File: react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx [hunk 2]
```diff
@@ -76,6 +81,9 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS }: IPro
...
+        }, NOTIFICATION_TIMEOUT_TYPE.LONG));
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx:86]: duration should be short.

File: react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx [hunk 3]
```diff
@@ -89,6 +97,9 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS }: IPro
     function _onCopyTextKeyPress(e: React.KeyboardEvent) {
         if (e.key === ' ' || e.key === 'Enter') {
             e.preventDefault();
+            dispatch(showSuccessNotification({
+                titleKey: "Copied meeting link"
```
Comment (saghul) [inline react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx:101]: Same as above.

**Review by saghul — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute! I left some comments, please take a look and check the linter for potential issues.

**Comment by AnshuPathak-88825** [comment]
for this titleKey: "Copied meeting link" I've to add this  "CopiedMeeting":"Copied meeting link" in every language main.json file ?
@saghul

**Comment by saghul** [comment]
You only need to add it to main.json, not to the other languages. Unless you happen to know how to speak any of them, in that case the translation is more than welcome :-)

**Comment by AnshuPathak-88825** [comment]
Done require  changes made 
Can I raise issue related for  main-hi.json there are  some json value  are in english I want to convert them in hindi ?
image

**Comment by damencho** [comment]
> Done require changes made Can I raise issue related for main-hi.json there are some json value are in english I want to convert them in hindi ? image

Just create a separate PR for those.

**Comment by damencho** [comment]
You need a space after the colon in main.json.

**Comment by AnshuPathak-88825** [comment]
done pls check @damencho

**Comment by saghul** [comment]
Please check the linter errors.

**Comment by mdrazak2001** [comment]
@AnshuPathak-88825: try running `npm run lint:ci -- --fix` this will fix your linter errors automatically.

**Review by saghul — APPROVED** [review]

File: lang/main.json [hunk 4]
```diff
@@ -304,6 +304,7 @@
         "connecting": "Connecting",
         "contactSupport": "Contact support",
         "copied": "Copied",
+        "copiedMeeting": "Copied meeting link",
```
Comment (saghul) [inline lang/main.json:307]: Hum, this is bad english. But also, why not just use the "Copied" text?

Comment (AnshuPathak-88825) [reply lang/main.json:307]: that's also fine I'll change that to "copiedMeeting": "Copied", this ?

**Comment by AnshuPathak-88825** [comment]
All changes made hope you'll review it soon @saghul

File: lang/main.json (hunk 4 above)
Comment (saghul) [reply lang/main.json]: Don't add a new key, just used the existing one please.

**Comment by damencho** [comment]
jenkins test this please

**Comment by damencho** [comment]
Jenkins test this please.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Thank you very much @AnshuPathak-88825 !

**Pull request merged**